troposphere/static/js/actions/tag/create.js

```diff
@@ -16,7 +16,8 @@ export default {
 
         var tag = new Tag({
             name: name,
-            description: description
+            description: description,
+            allow_access: name !== "featured",
```

Comment:
From using the Django REST Framework (DRF) reference, it looks like `"featured"` is `"Featured"`:

https://atmo.cyverse.org/api/v2/tags/515e396e-013d-42de-94f9-9e6d103bdcfa

If you consider a "featured" image, we can use the DRF to search for one, and inspect the `name` attribute:

https://atmo.cyverse.org/api/v2/images/12b10acb-b1f8-4d94-a56d-71cdfec7a856

So it looks like this might need to be `name !== "Featured"`. If we had to handle both, something like `/featured/i.test(name)` would give us a Boolean indicating if `name` matched the regular expression.